docs/source/api/tracker.md

```diff
@@ -320,9 +320,9 @@ await Tracker.autorun(async () => {
 
 ```
 
-For a better developer experience `firstRunPromise` is automagically appended to your async `autorun` calls so you don't have to write them yourself. Meaning this also works: 
```

Comment:
`automagically`, 🤔 I really like this word. Probably a typo here, but I think this should be something we should strive for in Meteor.

Reply:
It happened only twice in our code base, and it does not follow that much how we write our docs, but I agree. We could review our documentation guide line and make some adjustments to be more pleasing to read.

I like this writing in docs(Puns and [funny comments](https://github.com/meteor/meteor/pull/12824/commits/cc07fe5d03b1093f15c4214e9363bc37499c2404)) generally, but I need to maintain how it is today.

Reply:
It's not a typo. I meant to use it since Meteor is all about that magic. Pffffft you guys won't get it!